python/servo/package_commands.py

```diff
@@ -674,6 +677,22 @@ def upload_to_s3(platform, package, timestamp):
                 }
             )
 
+            if github_release_id:
+                from github import Github
+                g = Github(os.environ['NIGHTLY_REPO_TOKEN'])
+                nightly_repo = g.get_repo(os.environ['NIGHTLY_REPO'])
+                release = nightly_repo.get_release(int(github_release_id))
```

Comment:
I'm fairly sure the `int(...)` cast here can be avoided by specifying `type=int` in the `@CommandArgument` decorator.